src/operators/infinitepepo.jl

```diff
@@ -171,6 +171,32 @@ function InfiniteSquareNetwork(top::InfinitePEPS, mid::InfinitePEPO, bot::Infini
     )
 end
 
+function InfiniteSquareNetwork(mid::InfinitePEPO)
+    return InfiniteSquareNetwork(map(tuple, eachslice(unitcell(mid); dims=3)...))
+end
+
+"""
+    _dag(O::PEPOTensor)
+
+Calculate the conjugate of an operator O, while permuting the physical indices.
+Flips and twists are included to ensure the correct arrow convention of a PEPOTensor.
+"""
+
```

Comment:
```suggestion
```
Docstrings really need to be attached to function signatures without blank lines in between.